### DISCUSSION TITLE
How do I use {{if ...}} to test strings are equal?

### DISCUSSION THREAD

**Discussion by dave42w** [sunng87/handlebars-rust#566] [Q&A] [answered]
Hi,

In an html form I have a select box. I want to have the option that comes from the db be SELECTED

What I have that doesn't render is:

```
         <label for="name">Song Book:</label> <select name="song_book_id" default>
            {{#each song_book}}
              <option value="{{id}}" {{if id = ../song.song_book_id}}SELECTED{{/if}}>{{code}}:{{title}}</option>
            {{/each}}
          </select>
```

So in the option value I want to show SELECTED if the current song_book.id is equal to ../song.song_book_id

Is there a {{if syntax for this? Do I need a helper that can return a bool? 

thanks

**Comment by sunng87** [comment, marked as answer]
this is a helper `eq` https://docs.rs/handlebars/latest/handlebars/#built-in-helpers for this

try

```hbs
{{#if (eq id ../song.song_book_id)}} SELECTED {{/if}}
```

**Reply by dave42w** [reply]
Thanks. Just what I needed and it would be great to see this added to the docs. So far I've only seen isActive types used with {{#if
